src/potrf.cc

```diff
@@ -269,10 +268,11 @@ void potrf(
                                  depend(inout:column[j])
                 {
                     // A(j, j) -= A(j, k) * A(j, k)^H
+                    int queue_jk1 = j-k+2;
```

Comment:
Pedantic: Should the name be `queue_jk2` instead?

Reply:
Oh, yes! Thanks.